src/Administration/Resources/app/administration/src/module/sw-bulk-edit/page/sw-bulk-edit-product/index.js

```diff
@@ -359,7 +359,7 @@ export default {
                     type: 'int',
                     canInherit: false,
                     config: {
-                        componentName: 'mt-number-field',
+                        componentName: 'sw-number-field',
```

Comment:
We don't want to use the `sw-number-field` anymore. It is deprecated. The handling should be fixed inside the bulk edit

Reply:
Thank you 💙  I found another new way and just updated it :)